### DISCUSSION TITLE
stable版本云编译报错

### DISCUSSION THREAD

**Discussion by flyfishes** [Boos4721/openwrt#153] [Bugs/问题反馈]
用GitHub action 自动云编译：

**Compile firmware这一步骤显示3个错误：**

Overriding core package 'xray-plugin' with version from Boos
tmp/.config-package.in:107206:error: recursive dependency detected!
tmp/.config-package.in:107206:	symbol PACKAGE_wrtbwmon depends on BUSYBOX_DEFAULT_IP
package/utils/busybox/Config-defaults.in:2406:	symbol BUSYBOX_DEFAULT_IP depends on PACKAGE_busybox
tmp/.config-package.in:163:	symbol PACKAGE_busybox is selected by PACKAGE_wrtbwmon
For a resolution refer to Documentation/kbuild/kconfig-language.rst
subsection "Kconfig recursive dependency limitations"

 make[3] -C feeds/Boos/luci-app-unblockneteasemusic compile
 make[2] package/install
make -r world: build failed. Please re-run make with -j1 V=s or V=sc for a higher verbosity level to see what's going on
make: *** [/home/runner/work/openwrt/openwrt/include/toplevel.mk:230: world] Error 1
make[1]: Entering directory '/home/runner/work/openwrt/openwrt'
make[2]: Entering directory '/home/runner/work/openwrt/openwrt'
make[2]: Entering directory '/home/runner/work/openwrt/openwrt'
rm -rf /home/runner/work/openwrt/openwrt/staging_dir/target-aarch64_cortex-a53_musl/root-ipq807x
make[3]: Entering directory '/home/runner/work/openwrt/openwrt/target/linux'
make[4]: Entering directory '/home/runner/work/openwrt/openwrt/target/linux/ipq807x'
make[2]: Leaving directory '/home/runner/work/openwrt/openwrt'

Configuring luci-i18n-base-zh-cn.
make[2]: *** [package/Makefile:70: package/install] Error 255
Configuring luci-i18n-adguardhome-zh-cn.
make[1]: *** [package/Makefile:111: /home/runner/work/openwrt/openwrt/staging_dir/target-aarch64_cortex-a53_musl/stamp/.package_install] Error 2
Configuring bind-dig.
Configuring kmod-ath11k-ahb.
Configuring luci-app-pushbot.
Configuring kmod-lib-lzo.
Configuring luci-i18n-upnp-zh-cn.
Configuring luci-i18n-firewall-zh-cn.
Configuring ppp-mod-pppoe.
make[2]: Leaving directory '/home/runner/work/openwrt/openwrt'
make[1]: Leaving directory '/home/runner/work/openwrt/openwrt'
make: *** [/home/runner/work/openwrt/openwrt/include/toplevel.mk:230: world] Error 2
Build completed in 334 minute(s) and 0 seconds

**最后停止在Prepare artifact**
mkdir -p ./artifact/firmware
  mkdir -p ./artifact/package
  rm -rf $(find ./bin/targets/ -type d -name "packages")
  rm -rf $(find ./bin/targets/ -type f -name "*.buildinfo")
  cp -rf $(find ./bin/targets/ -type f) ./artifact/firmware/
  cp -rf $(find ./bin/packages/ -type f -name "*.ipk") ./artifact/package/
  shell: /usr/bin/bash -e {0}
  env:
    TZ: Asia/Shanghai
cp: missing destination file operand after './artifact/firmware/'
Try 'cp --help' for more information.
<img width="741" alt="微信截图_20220624111553" src="https://user-images.githubusercontent.com/34975488/175455113-578ff5fe-ab20-445f-8bc6-89c9625ac76d.png">

Error: Process completed with exit code 1.

**Comment by Boos4721** [comment]
<img width="478" alt="image" src="https://user-images.githubusercontent.com/43845654/175824238-532ca8e6-01c7-423e-b5c2-4b45e49e7ebc.png">
humm compile done.

**Reply by flyfishes** [reply]
编译是完成了，那Prepare artifact报错是什么原因呢？
